### PR TITLE
add start of deployment files

### PR DESCRIPTION
**Pull request by pixie79**
Adds drone, Dockerfile and kube files for deployment

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      event: push
+
+  unit_tests:
+    image: quay.io/ukhomeofficedigital/drone-docker
+    commands:
+      - docker run -t cop-data-api:$${DRONE_COMMIT_SHA} sh -c "npm run test"
+    when:
+      event: push
+
+  # lint_checks:
```
Comment (p3dr0migue1) [inline .drone.yml]: uhh erm...commented code? could we remove this :grimacing: please

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      - DEV_KUBE_TOKEN
+      - DEV_LOG_LEVEL
+      - DEV_DB_CONNECTION_STRING
+    commands:
+      - export HOSTNAME=$${DEV_HOSTNAME}
+      - export KUBE_NAMESPACE=$${DEV_KUBE_NAMESPACE}
+      - export KUBE_SERVER=$${DEV_KUBE_SERVER}
+      - export KUBE_TOKEN=$${DEV_KUBE_TOKEN}
+      - export LOG_LEVEL=$${DEV_LOG_LEVEL}
+      - export DB_CONNECTION_STRING=$${DEV_DB_CONNECTION_STRING}
```
Comment (p3dr0migue1) [inline .drone.yml]: `DB_CONNECTION_STRING` someone has actually read my code ;) :+1:

File: healthcheck.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  if (response && response.statusCode === 200) {
+    // eslint-disable-next-line no-console
+    console.log(`STATUS: ${response.statusCode}`);
+    process.exit(0);
+  } else {
+    // eslint-disable-next-line no-console
+    console.log(`ERROR: ${error}`);
+    process.exit(1);
+  }
+});
```
Comment (p3dr0migue1) [inline healthcheck.js]: looks familiar :joy: well done, nice

File: .drone.yml (hunk 1 above)
Comment (pixie79) [reply .drone.yml]: no - it is for you to add the linting :)

Comment (tomfitzherbert) [reply .drone.yml]: Linting step removed

**Comment by p3dr0migue1** [comment]
Very happy with changes

**Pull request merged**